jq.yaml

```
@@ -43,5 +40,8 @@ subpackages:
 
 update:
   enabled: true
-  release-monitor:
-    identifier: 13252
+  github:
+    identifier: stedolan/jq
```

Should this be updated?

```suggestion
    identifier: jqlang/jq
```